calblueprint/ogsc
# Updating the Invitations Requests Table

# Invitations Requests Table

- Fixing the Invitations Requests Table to accurately display requested user accounts only (vs all users, which was being displayed before)
- Added `emailVerified = new Date()` for all user creations except in the user account request/signup flow
- Renamed the `admin/users/readVerified` endpoint to `admin/users/accountRequests` + updated a few things to get it fetching the data

## TODO

- [ ] Should probably implement pagination? If so, I need to ask Shannon how that works
- [ ] 'Your Pending Invites' table next

## How to Test/Use PR feature

- Sign up for an account at http://localhost:3000/users/signUp
- Go to http://localhost:3000/admin/invite and you should see your info under the Invitations Request table

## PR Checklist

Does your branch (check if true):
_getting an unused `req` var in the endpoint, need pagination (if we don't need pagination then idk how to fix this)_


## Screenshots
Table no longer displays all users, only users who have signed up/requested for an account
<img width="1435" alt="Screen Shot 2021-02-01 at 10 04 40 PM" src="https://user-images.githubusercontent.com/34170094/106559667-93305b00-64da-11eb-9d1a-4c08150e5a3f.png">

CC: @sydbui
